admin/src/app/Dtos/room.ts

```diff
@@ -0,0 +1,9 @@
+export interface Room {
+  id: number;
+  name: string;
+  sleeps: number;
+  number: number;
+  price: number;
+  facilities?: string;
```

Comment:
`string`?

Reply:
Gonna change for Facility[] in the next pr. Right now it's just dummy for html